plugins/org.eclipse.reddeer.core/src/org/eclipse/reddeer/core/condition/ActiveShellExists.java

```diff
@@ -31,7 +31,7 @@
 	@Override
 	public boolean test() {
 		shell = ShellLookup.getInstance().getCurrentActiveShell();
-		return shell != null;
+		return shell != null ? true : false;
```

Comment:
there is no difference between shell != null and shell != null ? true : false. 

IMHO we have nothing to do here. We simply call SWT API and when it returns null active shell than we do not have a way to find out if really no shell is active